PinpointKit/PinpointKit/Sources/Screenshotter.swift

```diff
@@ -28,7 +28,10 @@ public class Screenshotter {
             window.drawViewHierarchyInRect(window.bounds, afterScreenUpdates: false)
         }
         
-        let image = UIGraphicsGetImageFromCurrentImageContext()
+        guard let image = UIGraphicsGetImageFromCurrentImageContext() else {
+            preconditionFailure("`UIGraphicsGetImageFromCurrentImageContext()` should never return `nil` as we satisify the requirements of having a bitmap-based current context created with `UIGraphicsBeginImageContextWithOptions(_:_:_:)`")
```

Comment:
We could instead propagate optionals all the way up, but the docs are pretty clear about this:

> Returns an image based on the contents of the current bitmap-based graphics context.
You should call this function only when a bitmap-based graphics context is the current graphics context. If the current context is `nil` or was not created by a call to `UIGraphicsBeginImageContext(_:)`, this function returns `nil`.